Frameworks/CoreText/CTLine.mm

```diff
@@ -38,7 +44,7 @@ - (instancetype)copyWithZone:(NSZone*)zone {
     ret->_ascent = _ascent;
     ret->_descent = _descent;
     ret->_leading = _leading;
-    ret->_runs = [_runs copy];
```

Comment:
if you need a mutable array, use mutableCopy

---
In reply to: [78225046](https://github.com/Microsoft/WinObjC/pull/989#discussion_r78225046) [](ancestors = 78225046)